src/sentry/static/sentry/app/views/eventsV2/table/tableActions.tsx

```diff
@@ -90,6 +90,15 @@ renderAsyncExportButton.propTypes = {
   isLoading: PropTypes.bool,
 };
 
+function renderSummaryButton() {
+  return (
+    <HeaderButton>
+      <IconTag size="xs" />
+      {t('Show Tag Summary')}
```

Comment:
Should this say toggle? Is there a use for showing and then hiding the tag summary?